clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx

```diff
@@ -30,105 +35,151 @@ const CreateBloodRequest = () => {
-            label='Blood Group'
-            options={bloodGroupOptions}
-            name='neededBloodGroup'
-            value={bloodRequestData.neededBloodGroup}
-            onChangeText={handleInputChange}
-            isVisible={isVisible}
-            setIsVisible={setIsVisible}
-            error={errors.neededBloodGroup}
-            isRequired={true}
-            readOnly={isUpdating}
+            extraInfo='Select "urgent" if the blood is needed on the same day'
           />
 
-          <CustomDropdown
-            placeholder=''
-            name='bloodQuantity'
-            label='Unit'
-            value={bloodRequestData.bloodQuantity}
-            onChangeText={handleInputChange}
-            options={bloodBagOptions}
-            isVisible={isVisible}
-            setIsVisible={setIsVisible}
-            error={errors.bloodQuantity}
-            isRequired={true}
-          />
-          <DateTimePickerComponent
-            label="Time and Date"
-            value={new Date(bloodRequestData.donationDateTime)}
-            onChange={(date) => handleInputChange(DONATION_DATE_TIME_INPUT_NAME, date)}
-            showDatePicker={showDatePicker}
-            setShowDatePicker={setShowDatePicker}
-            error={errors.donationDateTime}
-            isRequired={true}
-          />
-          <Input
-            name="location"
-            label="Location"
-            value={bloodRequestData.location}
-            onChangeText={handleInputChange}
-            placeholder="Search Location"
-            keyboardType='default'
-            error={errors.location}
-            isRequired={true}
-            readOnly={isUpdating}
-          />
-          <Input
-            name="contactNumber"
-            label="Contact Number"
-            value={bloodRequestData.contactNumber}
-            onChangeText={handleInputChange}
-            placeholder="Enter contact number"
-            keyboardType="phone-pad"
-            error={errors.contactNumber}
-            isRequired={true}
-          />
+          <View style={styles.fieldSpacing}>
+            <Dropdown
+              label='Blood Group'
+              isRequired={true}
+              placeholder='Select Blood Group'
+              options={bloodGroupOptions}
+              readonly={isUpdating}
+              name='neededBloodGroup'
+              selectedValue={bloodRequestData.neededBloodGroup}
+              onChange={handleInputChange}
+              error={errors.neededBloodGroup}
+            />
+          </View>
 
-          <Input
-            name="patientName"
-            label="Name of the Patient"
-            value={bloodRequestData.patientName}
-            onChangeText={handleInputChange}
-            placeholder="Enter patient's name"
-            keyboardType="twitter"
-            error=''
-          />
+          <View style={styles.fieldSpacing}>
+            <Dropdown
+              label='Unit'
+              isRequired={true}
```

Comment:
same

Reply:
Yes Vai, The purpose of `isRequired` is to specify whether the input field is mandatory or not.